package/libnetwork/src/libnetwork.c

```diff
@@ -928,14 +928,19 @@ static struct curl_slist *ntrip_init(network_context_t *ctx, CURL *curl)
   chunk = curl_slist_append(chunk, "Ntrip-Version: Ntrip/2.0");
   chunk = curl_slist_append(chunk, "Expect:");
 
+  while (!device_has_gps_time()) {
+    sleep(1);
```

Comment:
Is there any case in which we would not want to block indefinitely on gps time? Possibly for ntrip casters that don't require gga up?

Reply:
I was also a little worried that we are blocking indefinitely... at the very least, I think we could shorten this sleep up to 200ms or so.  Maybe we should put an upper limit on the time-out here, so that we'll work for 98% of "normal" cases, but not wait indefinitely?   I think a limit of 120s would be reasonable, given our current TTFF is around 60-70s.

Reply:
The RTCM decoder will not do anything before it is given gps time, so there's not much use connecting to a NTRIP stream before that. But I guess it won't hurt either.

Reply:
180s is commonly used in TTFF testing although I think it could be even longer if the user is having trouble with antenna or something else with his first setup. To match it better with 200ms sleep I compromised to 200s time limit.